doc/testing_guide.md

```diff
@@ -195,6 +195,12 @@ Please note that the benchmarking process is not ready for the PoC code yet and
 
 Benchmarking and updating weights should be done each time a new extrinsic is added to any pallet (weights are used to estimate transaction fees). Weights are obligatory for extrinsics that are available for users.
 
+To create a node with benchmarking capabilities you need to compile it with:
+```bash
+cargo build --release --features runtime-benchmarks
+```
+By default, nodes using in this Testing Guide does not support benchmarking.
+
```

Comment:
By default, nodes used in this Testing Guide do not support benchmarking.

Reply:
Done.